### PR TITLE
…

### PR DESCRIPTION
**Pull request by Vix2021**
This ticket does not have a JIRA as it was found whilst investigating a different issue.

If you set the Net Mode to "Play Standalone" and run the PIE session normally you get an exception in VS screenshot shown below, you can press continue and still play the game fine. 

image (10) (1)

The variable ‘GIsPlayInEditorWorld’  causes an exception, it is a global variable that is accessed by each PIE session. This is an Epic variable and the exception happens when not running Spatial OS.
If the setting “Run under one process” is disabled the exception does not occur so appears to be a multi-threading issue with the global variable and multiple windows.
This “Run under one process” option does have the warning that it can create issues.
The exception does not occur at all in the GDK Test Gyms. 
I’ve tracked down the trigger of the exception to the Example Project game instance “BP_GDKShooterGameInstance”. Without this game instance, the exception does not occur. This game instance inherits from “SpatialGameInstance” which works fine. It also only seems to duplicate existing functionality that is already in the base class. 
So the solution was to change the game instance in the example project to “SpatialGameInstance”. This will get rid of the exception without losing any functionality.

Exception details:
FirstHitException

BP_GDKShooterGameInstance blueprint:
GameInstanceClassBeforeDetails

Note the print messages on connecting to Spatial shown in the Blueprint are redundant as they are already in the base class SpatialGameInstance.

### CODE REVIEW

**Comment by m-samiec** [comment]
> Note the print messages on connecting to Spatial shown in the Blueprint are redundant as they are already in the base class SpatialGameInstance.

Does that mean the messages are double printed currently?

File: Game/Config/DefaultEngine.ini [hunk 1]
```diff
@@ -2,7 +2,7 @@
 GameDefaultMap=/Engine/Maps/Entry.Entry
 EditorStartupMap=/Game/Maps/Control_Small.Control_Small
 GlobalDefaultGameMode=/Game/GameMode/BP_ControlGameMode.BP_ControlGameMode_C
-GameInstanceClass=/Game/GameMode/BP_GDKShooterGameInstance.BP_GDKShooterGameInstance_C
```
Comment (m-samiec) [inline Game/Config/DefaultEngine.ini:5]: So BP_GDKShooterGameInstance isn't used at all anymore? Can we delete the asset?

**Comment by Vix2021** [comment]
> > Note the print messages on connecting to Spatial shown in the Blueprint are redundant as they are already in the base class SpatialGameInstance.
> 
> Does that mean the messages are double printed currently?

In "Net Mode" set to "Play As Client" yes the logs are duplicated:
```
LogSpatialGameInstance: Successfully connected to SpatialOS
LogBlueprintUserMessages: [BP_GDKShooterGameInstance_C_4] Client 0: Successfully connected to SpatialOS
```

However, in "Net Mode" set to "Play Standalone" no logs are output.

File: Game/Config/DefaultEngine.ini (hunk 1 above)
Comment (Vix2021) [reply Game/Config/DefaultEngine.ini:5]: Good point. I have now deleted the asset.

**Review by m-samiec — APPROVED** [review]

**Pull request merged**